### PR TITLE
Update p4v

### PR DESCRIPTION
**Pull request by mystygage**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/p4v.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask "p4v" do
   version "20.3,2059355"
```
Comment (michalsvec) [inline Casks/p4v.rb:2]: the version changed as well: 

```  
version "20.3,2060285"
sha256 "8b42a0e1dfe13da490a52bf03f65947b48b58a28bedb6277a1caf3a320a596c3"
```

**Review by michalsvec — CHANGES_REQUESTED** [review]

File: Casks/p4v.rb (hunk 1 above)
Comment (mystygage) [reply Casks/p4v.rb:2]: Sorry, I didn't double checked the version, I trusted the latest information on the website :/
image

Comment (michalsvec) [reply Casks/p4v.rb]: I'd do the same, but over the years I learned to not trust perforce :)

**Comment by mystygage** [comment]
PR can be closed, #97299 was faster

**Pull request closed without merging**